### PR TITLE
closes #11

### LINKED ISSUE #11: Demo for validation of classification including feature selection
If the classification method includes feature selection, that has to be performed on the training set only. In the old svn toolbox, there is a demo that first shows the wrong validation (feature selection on the whole data set) which underestimates the generalization error. In a second part, the valid validation is demonstrated. That should be transferred to the new git toolbox. Please use some of the demo datasets (see other demos).

### PR DESCRIPTION
**Pull request by MarkusWenzel**
Demo for validation of classification including feature selection

### CODE REVIEW

**Comment by BenjaminBlankertz** [comment]
I think the demo should be changed.
- The sample and the loss functions should be called directly without storing their handles in a variable.
- Use proc_flaten on fv instead of explicit reshaping.
- Add a proc_ function for feature selection, e.g., transfer proc_fs_FisherCriterion from the old toolbox
- Demonstrate correct validation in two ways: (a) performing feature selection via the 'proc' of crossvalidation (I hope that works), and (b) explicit cross-validation as now. See also the demo_feature_selection.m of the old toolbox.
